api/app/controllers/spree/api/checkouts_controller.rb

```diff
@@ -52,7 +52,7 @@ def update
         authorize! :update, @order, order_token
 
         if @order.update_from_params(params, permitted_checkout_attributes, request.headers.env)
-          if current_api_user.has_spree_role?('admin') && user_id.present?
+          if can?(:admin, @order) && user_id.present?
```

Comment:
just checking - does the owner of an order have the ability to admin an order? if so, could they change the user_id? that would be bad